CHANGELOG.en-US.md

```diff
@@ -15,6 +15,33 @@ timeline: true
 
 ---
 
+# 3.2.0
+
+`2018-02-04`
+
+- 🌟 Add new `tabBarGutter` prop to Tab to allow setting gutter between tabs. [#8644](https://github.com/ant-design/ant-design/pull/8644) [@lixiaoyang1992](https://github.com/lixiaoyang1992)
+- 🌟 Add new `hasSider` prop to Layout to avoid render error when server rendering. [#8937](https://github.com/ant-design/ant-design/issues/8937)
+- 🌟 Add new `successPercent` to Progess to alow showing two phases of progess. [Demo](https://ant.design/components/progress/#components-progress-demo-segment)
+- 🌟 Add new `iconType` prop to Alert to allow settting type of icon. [#8811](https://github.com/ant-design/ant-design/pull/8811) [@minwe](https://github.com/ant-design/ant-design/pull/8811)
+- 🌟 Add `id` propm to DatePicker. [#8598](https://github.com/ant-design/ant-design/pull/8598) [@mgrdevport](https://github.com/mgrdevport)
+- 🌟 Add new `forceRender` prop to Collapse to allow rendering hide panel. [#9192](https://github.com/ant-design/ant-design/pull/9192) [#Pyroboomka](https://github.com/ant-design/ant-design/pull/9192) [@paulcmason](https://github.com/react-component/collapse/pull/82)
+- RangePicker
+  - 🌟 Improve `ranges` prop to allow passig function to it. [#8281](https://github.com/ant-design/ant-design/issues/8281)
+  - 🐞 Fix issue resulting in can not input start dated manually. [#6999](https://github.com/ant-design/ant-design/issues/6999)
+  - 🐞 Fix issue resulting date panel closed without animation when click on the preset range. [#6364](https://github.com/ant-design/ant-design/issues/6364)
+  - 🐞 Fix issue resulting `onOk` is not trigged, when click on the preset range. [#7747](https://github.com/ant-design/ant-design/issues/7747)
+- Select
+  - 🌟 Improve `onChange`、`onDeselect`, they will receive selected `Option` as second paramteter.
+  - 🐞 Fix issue resulting in `onSelect` is not trigged when using automatic tokenization. [#9094](https://github.com/ant-design/ant-design/issues/9094)
+  - 🐞 Fix the missing scrollbar in Chrome.
+- 🌟 Improve Table's `rowSelection[getCheckboxProps]` prop, now the all of returns of `getCheckboxProps` will pass to checkbox. [#9054](https://github.com/ant-design/ant-design/pull/9054) [@mgrdevport](https://github.com/ant-design/ant-design/pull/9054)
```

Comment:
I'm not sure about this one : 

now the all of returns of `getCheckboxProps` will pass to checkbox -> now all the properties returned by `getCheckboxProps` will be passed to checkbox